### PR TITLE
Story 14 delete existing

### PR DESCRIPTION
**Pull request by Shriyanshagro**
This contains documentation and tests for #49 
This PR also contains documentation update w.r.t #48

### CODE REVIEW

File: src/collective/ifttt/tests/test_manage_trigger.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        self.view()
+        storage = getUtility(IRuleStorage)
+
+        # check that rule has been successfully created
+        self.assertEqual(1, len(storage.values()))
+
+        # test ifttt_manage interface
+        self.request.form = {}
+        # ERROR: storage value is not reached to manager
+        self.request.form['form.widgets.ifttt_triggers'] = [
```
Comment (Shriyanshagro) [inline src/collective/ifttt/tests/test_manage_trigger.py]: @datakurre self.request.form doesn't carry rule instance to IFTTTmanager interface while binded here in form's request.
Kindly suggest an alternative

Comment (datakurre) [reply src/collective/ifttt/tests/test_manage_trigger.py]: @Shriyanshagro With `self.request.form ...` we simulate form post from browser. In this context the value for `form.widgets.ifttt_triggers` should be a `token` from `availableTriggers` source https://github.com/collective/collective.ifttt/blob/master/src/collective/ifttt/browser/views.py#L37

Comment (Shriyanshagro) [reply src/collective/ifttt/tests/test_manage_trigger.py]: @datakurre Can you elaborate this more? I'm not able to call tokens from availableTriggers

Comment (datakurre) [reply src/collective/ifttt/tests/test_manage_trigger.py]: I assume this one https://github.com/collective/collective.ifttt/blob/1158ae61dd599d6143f2609d91c4d9a03ac36563/src/collective/ifttt/browser/views.py#L55

**Pull request merged**